ios/FluentUI/Bottom Sheet/BottomCommandingController.swift

```diff
@@ -0,0 +1,592 @@
+    }
+
+    private func removeBinding(_ binding: ItemBinding) {
+        itemToBindingMap.removeValue(forKey: binding.item)
+        viewToBindingMap.removeValue(forKey: binding.view)
+    }
+
+    private func clearAllItemViews(in location: ItemLocation) {
+        switch location {
+        case .heroSet:
+            heroItems.forEach {
+                if let binding = itemToBindingMap[$0] {
+                    removeBinding(binding)
+                }
+            }
+            heroCommandStack.arrangedSubviews.forEach { $0.removeFromSuperview() }
+        case .list:
+            expandedListSections.forEach {
+                $0.items.forEach {
+                    if let binding = itemToBindingMap[$0] {
+                        removeBinding(binding)
+                    }
+                }
+            }
+        }
+    }
+
+    private func createAndBindHeroCommandView(with item: CommandingItem) -> UIView {
+        let tabItem = TabBarItem(title: item.title, image: item.image, selectedImage: item.selectedImage)
+        let itemView = TabBarItemView(item: tabItem, showsTitle: true)
+        itemView.alwaysShowTitleBelowImage = true
+        itemView.numberOfTitleLines = 1
+        itemView.isSelected = item.isOn
+        itemView.accessibilityTraits.insert(.button)
+
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleHeroCommandTap(_:)))
+        itemView.addGestureRecognizer(tapGesture)
+
+        NSLayoutConstraint.activate([
+            itemView.heightAnchor.constraint(equalToConstant: Constants.heroButtonHeight)
+        ])
+        let widthConstraint = itemView.widthAnchor.constraint(equalToConstant: Constants.heroButtonWidth)
+        widthConstraint.isActive = !isInSheetMode
+
+        item.delegate = self
+        let binding = HeroItemBinding(item: item, view: itemView, location: .heroSet, widthConstraint: widthConstraint)
+        addBinding(binding)
+
+        return itemView
+    }
+
+    private func setupTableViewCell(_ cell: TableViewCell, with item: CommandingItem) {
+        let iconView = UIImageView(image: item.image)
+        iconView.tintColor = Constants.tableViewIconTintColor
+        cell.setup(title: item.title, subtitle: "", footer: "", customView: iconView, customAccessoryView: nil, accessoryType: .none)
+        cell.bottomSeparatorType = .none
+        cell.isEnabled = item.isEnabled
+        cell.backgroundColor = Constants.tableViewBackgroundColor
+    }
+
+    // Reloads view in place from the given item object
+    private func reloadView(from item: CommandingItem) {
+        guard let binding = itemToBindingMap[item] else {
+            return
+        }
+        let staleView = binding.view
+
+        switch binding.location {
+        case .heroSet:
+            if let stackIndex = heroCommandStack.arrangedSubviews.firstIndex(of: staleView) {
+                removeBinding(binding)
+                let newView = createAndBindHeroCommandView(with: item)
```

Comment:
In context, what is a "heroCommandView"? Is that wording very office specific / documented elsewhere in this repository? If not, I would like that documented somewhere to remove a little bit of an office-ism that is assumed knowledge =)